okonomiyaki/file_formats/_package_info.py

```diff
@@ -231,29 +255,69 @@ def to_string(self, metadata_version_info=MAX_SUPPORTED_VERSION):
         self._write_field(s, 'Home-page', self.home_page)
         self._write_field(s, 'Author', self.author)
         self._write_field(s, 'Author-email', self.author_email)
-        self._write_field(s, 'License', self.license)
+
+        # Maintainer
+        if metadata_version_info >= (1, 2):
+            if self.maintainer:
+                self._write_field(s, 'Maintainer', self.maintainer)
+            if self.maintainer_email:
+                self._write_field(s, 'Maintainer-email', self.maintainer_email)
+
+        # License
```

Comment:
I would think that it is clear from the next few lines that this comment is stating the obvious. Please remove such comments from the `to_string` method